relay-event-normalization/src/normalize/processor.rs

```diff
@@ -62,7 +81,7 @@ impl<'a> From<LightNormalizationConfig<'a>> for NormalizeProcessorConfig<'a> {
 /// The returned [`ProcessingResult`] indicates whether the passed event should
 /// be ingested or dropped.
 #[derive(Debug, Default)]
-pub struct NormalizeProcessor<'a> {
+pub(crate) struct NormalizeProcessor<'a> {
```

Comment:
same comment about `pub(crate)`

Reply:
Resolving as this is addressed in a follow-up PR: https://github.com/getsentry/relay/pull/2729.